pkgs/development/python-modules/hickle/default.nix

```diff
@@ -33,6 +33,12 @@ buildPythonPackage rec {
 
   pythonImportsCheck = [ "hickle" ];
 
+  disabledTestPaths = [
+    # test_scalar_compression broken in 5.0.2 with recent NumPy
+    # see https://github.com/telegraphic/hickle/issues/174
+    "hickle/tests/test_hickle.py"
```

Comment:
Probably we don't want to disable this whole file, but since this unbreaks the package I can also make a separate PR to address this.

Reply:
My mistake, didn't realize I can exclude individual tests. This is fixed now =)